main/kamailio/APKBUILD

```diff
@@ -3,18 +3,17 @@
 # Contributor: Leonardo Arena <rnalrd@gmail.com>
 # Maintainer: Nathan Angelacos <nangel@alpinelinux.org>
 pkgname=kamailio
+pkgver=5.0.3
+pkgrel=0
 
 # If building from a git snapshot, specify the gitcommit
-# run "abuild snapshot", then abuild as normal
 # If building a proper release, leave gitcommit blank
-#_gitcommit=e27802533cee
-_giturl="git://github.com/$pkgname/$pkgname.git"
-_gittag=HEAD
+#_gitcommit=39b9a1e0b05c05c50656588a863b564fd0708a6d
 
-
-pkgver=5.0.2
-pkgrel=2
-[ -z "${_gitcommit}" ] && _suffix="_src" || _suffix="-${_gitcommit}"
+[ ! -z "${_gitcommit}" ] && pkgver="${pkgver}.`date +%Y%m%d`"
+[ ! -z "${_gitcommit}" ] && pkgrel="${pkgrel}.${_gitcommit:0:7}"
```

Comment:
This is completely wrong, `pkgrel` is APKBUILD’s version, not the upstream’s package. Moreover it’s not even used here, so please just remove it, keep the abuild simple.